code-of-conduct.md

```diff
@@ -5,7 +5,7 @@ layout: default
 ## Be Awesome to Each Other
 
 The mission of [The London Django Meetup Group](http://www.djangolondon.com/)
-is to provide fun, welcoming and professional environments so that diverse
+is to provide a fun, welcoming and professional environments so that diverse
```

Comment:
undo this change - there shouldn't be an "a" because it's talking about multiple environment*s*